### ISSUE TITLE
Youtrack exception after update to latest version PHP storm

### ISSUE THREAD

**Issue by jk1** [jk1/youtrack-idea-plugin#111]
```
java.lang.Throwable: getComponent must be called on appropriate container (current: Project (name=inside, containerState=ACTIVE, componentStore=/home/joker/code/inside) , expected: Application (containerState=ACTIVE) )
	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:145)
	at com.intellij.serviceContainer.PlatformComponentManagerImpl.getComponent(PlatformComponentManagerImpl.kt:271)
	at com.github.jk1.ytplugin.ComponentAware$DefaultImpls.getIssueStoreComponent(ComponentAware.kt:41)
	at com.github.jk1.ytplugin.issues.IssueStoreUpdaterComponent.getIssueStoreComponent(IssueStoreUpdaterComponent.kt:15)
	at com.github.jk1.ytplugin.issues.IssueStoreUpdaterComponent$initComponent$1$1.run(IssueStoreUpdaterComponent.kt:27)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:906)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:779)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:422)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:698)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:421)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

**Comment by asamofal** [comment]
@jk1 thank you, It's my problem. Developers, if you need some more details I can provide it for you.

**Comment by jk1** [comment]
Released in scope of 0.14
